### DISCUSSION TITLE
Import Plugins

### DISCUSSION THREAD

**Discussion by ghost** [wwmm/easyeffects#2928] [Q&A] [answered]
Is it possible to install more plugins?  If so where do I drop them, and what plugin formats does it accept?  

I installed via dnf on fedora 39.

**Comment by violetmage** [comment, marked as answer]
If you mean effects, then no, it is not possible to add new effects like you would a VST/LV2 plugin. This is because easyeffects uses a custom GUI, where the plug-in's controls are reimplemented by hand in easyeffects' code.

Simply put, easyeffects is not supposed to be a generic plug-in host (like [carla](https://kx.studio/Applications:Carla)), but is instead intended to be an extremely easy to use, self-contained interface to a curated set of plug-ins.
